backend/src/handlers/PoolSnapshot.ts

```diff
@@ -237,12 +210,7 @@ export default async function ({
             Number(vaultRef.timestamp)
           )
           if (vaultAPY !== null) {
-            await context.db
-              .update(relayPool, {
-                chainId: pool.chainId,
-                contractAddress: pool.contractAddress,
-              })
-              .set({ apy: vaultAPY })
+            calculatedVaultAPY = vaultAPY
```

Comment:
I am confused because `vaultAPY` is also calculated. Why introduce a new var here?